packages/components/inputs/date-range-input/README.md

```diff
@@ -45,24 +45,53 @@ export default Example;
+| `onBlur`               | `Function`<br/>[See signature.](#signature-onBlur)                                           |          |         | Called when the date input loses focus.                                                                                                                                                                       |
+| `id`                   | `string`                                                                                     |          |         | Used as the HTML `id` attribute.                                                                                                                                                                              |
+| `name`                 | `string`                                                                                     |          |         | Used as the HTML `name` attribute.                                                                                                                                                                            |
+| `placeholder`          | `string`                                                                                     |          |         | Placeholder value to show in the input field                                                                                                                                                                  |
+| `isDisabled`           | `boolean`                                                                                    |          |         | Disables the date picker                                                                                                                                                                                      |
+| `isReadOnly`           | `boolean`                                                                                    |          |         | Disables the date picker menu and makes input field read-only                                                                                                                                                 |
+| `hasError`             | `boolean`                                                                                    |          |         | Indicates the input field has an error                                                                                                                                                                        |
+| `hasWarning`           | `boolean`                                                                                    |          |         | Indicates the input field has warning                                                                                                                                                                         |
+
+## Signatures
+
+### Signature `intl`
+
+```ts
+{
+  locale: string;
+  formatMessage: (message: MessageDescriptor) => void;
+}
+```
+
+### Signature `value`
```

Comment:
Is this still an issue in the generator?

Reply:
No. I needed to rerun the generator script.